it/config/DocumentationControllerISpec.scala

```diff
@@ -74,20 +75,42 @@ class DocumentationControllerISpec extends IntegrationBaseSpec {
   }
 
   "an OAS documentation request" must {
-    "return the documentation that passes OAS V3 parser" in {
-      val response: WSResponse = await(buildRequest("/api/conf/1.0/application.yaml").get())
-      response.status shouldBe Status.OK
+    List(Version1).foreach { version =>
+      s"return the documentation for $version" in {
+        val response = get(s"/api/conf/${version.name}/application.yaml")
+
+        val body         = response.body[String]
+        val parserResult = Try(new OpenAPIV3Parser().readContents(body))
+        parserResult.isSuccess shouldBe true
```

Comment:
just wondering if this could follow the same pattern as the "getOrElse(fail)" line below?